src/js/dropdown.ts

```diff
@@ -1,20 +1,51 @@
 import Choices from "choices.js";
 import "choices.js/public/assets/styles/choices.css";
 import scoreCardsData from "../../data/score-cards.json";
-import { CityId } from "./types";
+import { CityId, ScoreCardDetails } from "./types";
 
 export const DROPDOWN = new Choices("#city-choice", {
   allowHTML: false,
   itemSelectText: "Select",
   searchEnabled: true,
+  shouldSort: false, // since cities are already alphabetical order in scorecard, disabling this option allows us to show PRN maps at the top.
 });
 
+const typedScoreCardData: Record<string, ScoreCardDetails> = scoreCardsData;
+
 const setUpDropdown = (initialCityId: CityId, fallBackCityId: CityId) => {
-  const cities = Object.entries(scoreCardsData).map(([id, { name }]) => ({
-    value: id,
-    label: name,
-  }));
-  DROPDOWN.setChoices(cities);
+  const allCities = Object.entries(typedScoreCardData).map(
+    ([id, { name, contribution }]) => ({
+      value: id,
+      label: name,
+      contribution: contribution || "PRN",
+    })
+  );
+  const communityCities = allCities.filter(
+    (city) => city.contribution !== "PRN"
+  );
+  DROPDOWN.setChoices([
+    {
+      value: "Official Maps",
+      label: "Official Maps",
+      disabled: false,
+      choices: allCities.filter((city) => city.contribution === "PRN"),
+    },
+    {
+      value: "Community Maps",
+      label: "Community Maps",
+      disabled: false,
+      choices:
+        communityCities.length > 0
+          ? communityCities
+          : [
+              {
+                value: "",
+                label: "No community maps available",
+                disabled: true,
+              },
+            ],
+    },
```

Comment:
What about not showing this at all if there are no community maps?

Reply:
That makes sense. We'll just have the label for `Official Maps`. Makes testing easier too.